gputop-webui/gputop-ui.js

```diff
@@ -506,7 +517,11 @@ GputopUI.prototype.set_paused = function(paused) {
     if (metric === undefined)
         return;
 
-    this.update_metric_set_stream(metric);
+    var hw_id = this.current_hw_id;
+    if (hw_id === undefined)
+        return;
```

Comment:
So with this change, if no hw id is selected, we can't pause the stream?

Reply:
Let me briefly introduce my idea. If we create 2 vGPUs, there are 3 states. ctx_mode = ["Global", "vGPU1", "vGPU2"], and corresponding hw_id = ["0", hw_id1, hw_id2]. The "Global" state means the current state, without applying my patch, that is, it shows the whole counter values for any context id. All of the vgpu_id and hw_id starts from non-zero value. Hence I define the hw_id as 0 to map the "Global" state. So in the beginning without selecting any hw id, the current_hw_id is 0, it works in the "Global" state.       
As a result, we can pause the stream whether we select one hw id or not.